examples/07_advanced/plot_bids_analysis.py

```diff
@@ -69,7 +64,6 @@
     img_filters=[("desc", "preproc")],
     n_jobs=2,
     space_label="",
-    sub_labels=["01", "02", "05", "08"],  # comment to run all subjects
```

Comment:
unrelated but we had discussed previously to not run examples on a subsample of subjects unless it REALLY took too long to run.